### PR TITLE
:bug: Fixed Odd Terminal Behaviour

### LINKED ISSUE #131: Odd Terminal Behaviour
The following happens to me when I try to enter a new line after running cleansio
```
cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$ cleansio((HEAD detached at c9eeb2b))$
```

After going back through the history I found that opening a file without an extension causes the issue. @levin-noro it did matter haha.

### PR DESCRIPTION
**Pull request by PatrickDuncan**
Resolves #131

### CODE REVIEW

File: tests/utils/test_files.py [hunk 1]
```diff
@@ -13,3 +15,23 @@ def test_file_name_no_ext_with_slash():
 
 def test_file_name_no_ext_only_file():
     assert file_name_no_ext('file!!.mp3') == 'file!!'
+
+# append_before_ext
+
```
Comment (levin-noro) [inline tests/utils/test_files.py]: Maybe also add test cases for empty path in order to have more test coverage?

```
def test_append_before_ext_empty_string_empty_string():
    assert append_before_ext('', '') == ''

def test_append_before_ext_empty_string_empty_string():
    assert append_before_ext('', '-acc') == '-acc'
```
